accounts/cachedb_test.go

```diff
@@ -394,6 +397,23 @@ func TestAccountCache_CacheDB_SyncFS2DB(t *testing.T) {
 	ma = nil
 }
 
+func TestCacheDBFilePath (t *testing.T) {
+	dir := filepath.Join("testdata", "keystore")
+	dir, _ = filepath.Abs(dir)
+	cache := newCacheDB(dir)
+
+	accs := cache.accounts()
+
+	for _, a := range accs {
+		if filepath.IsAbs(a.File) {
+			t.Errorf("wanted relative filepath (wanted basename), got: %v", a.File)
+		}
+	}
+
+	cache.close()
+	cache = nil
```

Comment:
I was running into some bugs that I think (thought?) had to do with parallel tests and/or sharing the same cache dir. `cache.close()` seems to be enough now, will remove.